Gems/AtomLyIntegration/CommonFeatures/Code/CMakeLists.txt

```diff
@@ -115,6 +114,25 @@ if(PAL_TRAIT_BUILD_HOST_TOOLS)
             Legacy::EditorCommon
     )
 
+    ly_add_target(
+        NAME AtomLyIntegration_CommonFeatures.Editor GEM_MODULE
+        NAMESPACE Gem
+        FILES_CMAKE
+            atomlyintegration_commonfeatures_shared_files.cmake
```

Comment:
just so others know, this file was already there (it just has the Module file)

Reply:
Thanks for calling it out, come to think of it should I remove `Module.cpp` from `atomlyintegration_commonfeatures_editor_files.cmake` now?